ItemChanger.Core/Events/GameEvents.cs

```diff
@@ -6,6 +6,9 @@
 
 namespace ItemChanger.Events;
 
+/// <summary>
+/// Central dispatcher for scene and persistence events raised by ItemChanger.
```

Comment:
Rather than call out the specific events here it's probably better to call out the type of event - gameevents are meant for things that happen during gameplay/caused by the game itself as opposed to itemchanger startup/shutdown lifecycle